support/SubmitScripts/OceanClang.sh

```diff
@@ -112,23 +112,25 @@ if [[ $checkpoints == 0 ]]; then
   # If a checkpoint is found add one to checkpoint and sumbit next job
   if test -e "${PWD}/SpectreCheckpoint$current_checkpoint"; then
     cp ../OceanClang.sh .
-    sed -i '106 s/0/1/1' OceanClang.sh
+    sed -i 's/checkpoints=0/checkpoints=1/' OceanClang.sh
     sbatch OceanClang.sh
   fi
-# Section to start from checkpoints
+# Section to start from checkpoint
 elif [[ $checkpoints -gt 0 && $checkpoints -lt 1000000 ]]; then
   ln -s ${PWD}/../SpectreCheckpoint$current_checkpoint .
   charmrun ${SPECTRE_COMMAND} +restart SpectreCheckpoint$current_checkpoint \
-  --input-file ${SPECTRE_INPUT_FILE}
+          --input-file ${SPECTRE_INPUT_FILE}
   sleep 10s
   # If the next checkpoint was created modify variables and submit next job
-  next_checkpoint=$(python -c 'print(str('$checkpoints').zfill(6))')
+  printf -v next_checkpoint %06d $checkpoints
   if test -e "${PWD}/SpectreCheckpoint$next_checkpoint"; then
     cp ../OceanClang.sh .
     next_num_of_checkpoints=$(($checkpoints + 1))
     #Updating variables for the next possible checkpoint
-    sed -i '106 s/'$checkpoints'/'$next_num_of_checkpoints'/1' OceanClang.sh
-    sed -i '107 s/'$current_checkpoint'/'$next_checkpoint'/1' OceanClang.sh
+    sed -i 's/checkpoints='$checkpoints'/'\
```

Comment:
If you use double quotes you don't need to leave the quotes to expand variables, so `"s/checkpoints=$checkpoints/"`.  No functional difference in this case, but I think it's a bit easier to read.